sherpa/image/DS9.py

```diff
@@ -237,8 +237,30 @@ def __init__(self, *args, **kargs):
 
         if doRaise:
             raise RuntimeErr('badwin', _ex)
+
+    elif sys.version_info < (3, 2):
```

Comment:
An alternative to this (that is a bit-more Pythonic) is

    elif hasattr(subprocess.Popen, '__enter__'):
        _Popen = subprocess.Popen
    else:
        class _Popen(subprocess.Popen):
            ... define __enter__ and __exit__